repos/fdbt-site/src/pages/api/deleteOperatorGroup.ts

```diff
@@ -0,0 +1,67 @@
+import { NextApiResponse } from 'next';
+import {
+    // deleteOperatorGroupByNocCodeAndId,
+    getAllProductsByNoc,
+    getOperatorGroupByNocAndId,
+} from '../../data/auroradb';
+import { redirectToError, redirectTo, getAndValidateNoc } from '../../utils/apiUtils/index';
+import { NextApiRequestWithSession } from '../../interfaces';
+import { getProductsMatchingJson } from 'src/data/s3';
+// import { updateSessionAttribute } from 'src/utils/sessions';
+// import { MANAGE_OPERATOR_GROUP_ERRORS_ATTRIBUTE } from 'src/constants/attributes';
+
+export default async (req: NextApiRequestWithSession, res: NextApiResponse): Promise<void> => {
+    try {
+        const { query } = req;
+        const id = Number(query?.id);
+
+        if (!Number.isInteger(id)) {
+            throw new Error('insufficient data provided for delete query');
+        }
+        const nocCode = getAndValidateNoc(req, res);
+
+        const products = await getAllProductsByNoc(nocCode);
+        const matchingJsonLinks = products.map((product) => product.matchingJsonLink);
+
+        const tickets = await Promise.all(
+            matchingJsonLinks.map(async (link) => {
+                return await getProductsMatchingJson(link);
+            }),
+        );
+
+        const ticketsWithOperatorGroup = tickets.filter((t) => t.passengerType.id === id);
+        // const productNames = ticketsWithOperatorGroup
+        //     .map((ticket) =>
+        //         ticket.products[0]
+        //             ? 'productName' in ticket.products[0]
```

Comment:
why are we commenting this instead of removing it?

Reply:
it is a new page and it will help the other developer who will be working on it to uncomment and use the code 

It is commented as we don't have the netex part completed